lib/custom/zalgo.js

```diff
@@ -58,10 +58,10 @@ module['exports'] = function zalgo(text, options) {
   function heComes(text, options) {
     var result = '', counts, l;
     options = options || {};
-    options["up"] = options["up"] || true;
-    options["mid"] = options["mid"] || true;
-    options["down"] = options["down"] || true;
-    options["size"] = options["size"] || "maxi";
+    options["up"] =   typeof options["up"]   !== 'undefined' ? options["up"]   : true;
+    options["mid"] =  typeof options["mid"]  !== 'undefined' ? options["mid"]  : true;
+    options["down"] = typeof options["down"] !== 'undefined' ? options["down"] : true;
+    options["size"] = typeof options["size"] !== 'undefined' ? options["size"] : "maxi";
```

Comment:
Why are these 4 lines changed? The earlier code is valid. Please revert this back.